arch/arm/src/stm32/stm32_sdio.c

```diff
@@ -172,9 +172,14 @@
 #define SDIO_CMDTIMEOUT          (100000)
 #define SDIO_LONGTIMEOUT         (0x7fffffff)
 
-/* Big DTIMER setting */
+/* DTIMER setting
+ *
+ * Assuming Max timeout in bypass 48 Mhz
+ *
+ */
 
-#define SDIO_DTIMER_DATATIMEOUT  (0x000fffff)
+#define IP_CLCK_FREQ UINT32_C(48000000)
+#define SDIO_DTIMER_DATATIMEOUT_MS 250
```

Comment:
```suggestion
#define IP_CLCK_FREQ               UINT32_C(48000000)
#define SDIO_DTIMER_DATATIMEOUT_MS 250
```
nit: suggestion for alignment

Reply:
Reformatted